src/LocalDate.php

```diff
@@ -456,6 +456,12 @@ public function plusDays(int $days): LocalDate
             return $this;
         }
 
+        if ($days === 1) {
```

Comment:
Could you please add a comment to explain why this code is there?

```suggestion
        // Performance optimization for a common use case.
        if ($days === 1) {
```